core/common/src/grpc_module.rs

```diff
@@ -4,9 +4,9 @@
 
 use tonic::transport::server::RoutesBuilder;
 
-/// Trait that must be implemented for an extension to add a grpc service to the hosted server.
-/// Note: This trait may be renamed in the future.
-pub trait GrpcExtension {
+/// Trait that must be implemented for a module to add one or more grpc services to the hosted
```

Comment:
Should this description also mention layers/middleware?

Reply:
Updated